
Updated version of an older patch.  sys/ioccom.h is needed for _IOR/_IORW

diff -ur gst-plugins-good-1.18.5.orig/sys/v4l2/ext/videodev2.h gst-plugins-good-1.18.5/sys/v4l2/ext/videodev2.h
--- gst-plugins-good-1.18.5.orig/sys/v4l2/ext/videodev2.h	2021-09-08 14:02:51.183566800 +0000
+++ gst-plugins-good-1.18.5/sys/v4l2/ext/videodev2.h	2021-10-27 01:29:56.431243355 +0000
@@ -62,6 +62,10 @@
 #include <sys/ioctl.h>
 #endif
 
+#if defined(__sun)
+#include <sys/ioccom.h>
+#endif
+
 #include "ext/types-compat.h"
 #include "ext/v4l2-common.h"
 #include "ext/v4l2-controls.h"
